_projects/pr-metrics.md

```diff
@@ -0,0 +1,7 @@
+---
+title: PR Metrics
+logo: pr-metrics.png
+description: An Azure DevOps build task that augments Azure DevOps and GitHub pull request titles, letting reviewers quickly determine PR size and test coverage.
+linkText: Try PR Metrics
+projectUrl: https://marketplace.visualstudio.com/items?itemName=ms-omex.prmetrics
```

Comment:
I wonder if instead this should point to https://github.com/microsoft/OMEX-Azure-DevOps-Extensions vs the actual install URL... asking the team for input.

Reply:
I only set it like the Marketplace URL as other projects seemed to be doing similar things – e.g., [AccessibilityInsights](https://github.com/microsoft/opensource.microsoft.com/blob/main/_projects/accessibility-insights.md) or [Blazor](https://github.com/microsoft/opensource.microsoft.com/blob/main/_projects/blazor.md).

But those examples are slightly different, so I'm happy to switch to the GitHub URL if we believe it is preferable. Just let me know.

Reply:
My preference would be for the GitHub repo, the readme can point to Marketplace (etc).

Reply:
Makes sense, I've updated it now.